CMakeLists.txt

```diff
@@ -1,9 +1,11 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 2.8.1)
-PROJECT(dot11decrypt)
 
-FIND_PACKAGE(libtins REQUIRED)
+SET(CMAKE_CXX_STANDARD 11)
+SET(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+PROJECT(dot11decrypt)
 
-SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++0x")
+FIND_LIBRARY(TINS_LIBRARY tins)
```

Comment:
Why didn't `FIND_PACKAGE` work? libtins should be exporting a script to find it via CMake.

Reply:
I am using armbian 5.73 and for some reason find_package didn't work when I installed libtins-dev via apt.